R/redundant_equals_linter.R

```diff
@@ -30,6 +35,11 @@
 #'   linters = redundant_equals_linter()
 #' )
 #'
+#' lint(
+#'   text = "dt[!(x), y]",
```

Comment:
I prefer keeping `!` inside `()` myself:

```suggestion
#'   text = "dt[(!x), y]",
```

Reply:
I also think a comment is warranted here...